examples/cdf/definition.yml

```diff
@@ -17,16 +17,23 @@
 # Commented values are shown with the defaults, uncomment and change to override
 
 df:
-#  nodes_min: 3
-#  nodes_max: 5
+#  min_k8s_nodes: 3
+#  max_k8s_nodes: 5
 #  public_loadbalancer: True
-#  ip_ranges: []
+#  loadbalancer_ip_ranges: []
+#  kube_ip_ranges: []
+#  cluster_subnets: []
+#  loadbalancer_subnets: []
```

Comment:
Suggest we add 
terminate_deployments: True
as an example.

Reply:
I debate whether this should be defaulted... I know probably "not" because we can't yet deploy a flow with the cli (lame). But I expect that sort of usage to be common.

Reply:
Suggest we add 
df.teardown.persist
as well. I foresee lots of uncleaned up RDSs....
At least commented out with an explanation. It's not clear what "persists" (I believe it's the RDS).